docs/deployment/k8s/index.mdx

```diff
@@ -51,7 +55,7 @@ config:
     enabled: true
     fileStore:
       pvc:
-        size: 10Gi
+        size: 5Gi

```

Comment:
I'm not specifically commenting on the size change, I think this is great, but we might want to include enabling the websockets listener in this config block for future `wash ui` or `satellite-ui` examples, which would effectively just mean adding this:
```yaml
config:
...
  websocket:
    enabled: true
    port: 4223
```